portable/GCC/NiosII/port_asm.S

```diff
@@ -74,24 +74,13 @@ save_sp_to_pxCurrentTCB:
 	movia	et, pxCurrentTCB	# Load the address of the pxCurrentTCB pointer
 	ldw		et, (et)			# Load the value of the pxCurrentTCB pointer
 	stw		sp, (et)			# Store the stack pointer into the top of the TCB
-	
-	.section .exceptions.irqtest, "xa"	

```

Comment:
what's going on here?

Reply:
After `save_sp_to_pxCurrentTCB`,  jump to `.exceptions.irqtest` section. This `.exceptions.irqtest` is written in BSP generated `.S` file and this process will call irq hanlders if needed.